Dockerfile

```diff
@@ -24,6 +24,7 @@ COPY pyproject.toml poetry.lock /app/
 RUN /venv/bin/pip install poetry
 RUN /venv/bin/poetry install
 RUN /venv/bin/poetry add gunicorn
+RUN /venv/bin/poetry add loggly-python-handler
```

Comment:
Reminder that this line isn't necessary because it will already be installed by the `poetry install` step, since you've added the library outside of Docker and it's now recorded as part of your `pyproject.toml`/`poetry.lock` files